help/de/Help_ch_Account-Actions.xml

```diff
@@ -1,910 +1,1049 @@
+          Ertragskonten sein.
         </para>
       </listitem>
 
       <listitem>
-        <para>Stock: All accounts except Equity, Expense and Income can be child accounts.
+        <para>Devisenhandel: Unterkonto können alle Konten außer Eigenkapital-, Aufwands- und Ertragskonten
+          sein.
         </para>
       </listitem>
     </itemizedlist>
+ -->
   </sect1>
 
   <sect1 id="chart-create">
-    <title>Creating a Chart of Accounts</title>
-
-    <para>The Chart of Accounts is like a table of contents for your finances. The best way to conceptualize a
-      chart of accounts is as a tree. The main branches represent entire categories or groups, while
-      the leaves of the tree denote individual bank accounts or expense categories. When a summary
-      report is requested, typically only the main branches are shown in the report, rather than the
-      individual accounts. For example, a chart of accounts might look like the following:
+    <title>Neue Kontenhierarchie erstellen</title>
+
+    <para>Der Kontenplan ist das – hierarchisch angeordnete – Inhaltsverzeichnis ihrer Konten. Am besten
+      denkt man sich einen Kontenplan als Baum: Die Äste stehen für ganze Kategorien oder Gruppen,
+      während die Blätter einzelne Bankkonten oder Aufwandsarten bezeichnen. Wenn Sie einen
+      zusammenfassenden Bericht aufrufen, werden typischerweise nur die Hauptäste im Bericht
+      dargestellt und nicht die Einzelkonten. Eine Kontenhierarchie könnte beispielsweise wie folgt
+      aussehen:
     </para>
 
     <xi:include href="Help_chtacctseg.xml" />
 
-    <note id="accts-code">
-      <title>Accounts codes</title>
-
-      <para>Note that accounts not only have names; they may have codes, to order the accounts. When a report is
-        generated, the sort order is determined by the numbering. It&rsquo;s customary to have the
-        leaf accounts end in non-zero digits, while parent nodes have increasing numbers of zeros.
+    <tip id="accts-code">
+      <title>Kontennummer</title>
+<!-- FIXME source: Since it is difficult to visualise factual relationships with alphabetical sorting by account name, it is recommended to use and sort by alphanumeric account numbers. -->
+      <para>Da man mit der alphabetische Sortierung nach Kontonamen sachliche Zusammenhänge nur schwer
+        visualisieren kann, empfiehlt sich die Verwendung und Sortierung von alphanumerischen
+        Kontonummern. Wenn Sie einen Bericht erstellen, wird die Sortierreihenfolge nach der Nummer
+        bestimmt. Es ist üblich, dass die Nummern der <quote>Blatt</quote>-Konten nicht mit Null
+        enden, während die Nummern der höheren Hierarchieebenen jeweils mehr Nullen am Ende haben.
       </para>
-    </note>
+    </tip>
 
     <warning>
-      <para>&app; does not prevent duplicate numbering, although we would encourage you to avoid this. Account
-        codes are treated as numbers in base-36, thus, if you run out of numbers, you can use the
```

Comment:
Base-36 is no longer valid. You can enter almost any UTF character.But for a language independend sort order it's recommend to restrict it to a…z.